functions/_common/grapherTools.ts

```diff
@@ -16,6 +16,7 @@ import { ImageOptions } from "./imageOptions.js"
 
 interface FetchGrapherConfigResult {
     grapherConfig: GrapherInterface | null
+    multiDimDimensions?: string[]
```

Comment:
Maybe to make it even clearer what this array contains, rename it to `multiDimAvailableDimensions`?

Reply:
Done.